### PR TITLE
Add results of recent mRNA-1273 study

### PR DESCRIPTION
**Pull request by cgreene**
### Description of the proposed additions or changes

I updated the text to include the results of [@doi:10.1056/NEJMoa2022483]

### Related issues

#556 

### Suggested reviewers (optional)

@likhithakolla @esell17 @rmrussell @nilswellhausen 

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by rando2** [comment]
#554 didn't quite resolve this because there were a couple of places where there were TWO spaces at the end, and I didn't catch those. But for review, all of the actual changes begin on the line that says "RNA vaccines" (currently 805)

File: content/20.treatments.md [hunk 1]
```diff
@@ -450,13 +450,13 @@ The increased risk of thrombotic complications in COVID-19 infected patients was
...
 Based on concern among the medical community that the use of investigational therapeutics on COVID-19 patients already on antiplatelet therapies due to a pre-existing comorbidities may lead to issues with dosing and drug choice, and/or negative drug-drug interactions [@doi:10.1016/j.jacc.2020.04.031], this supplementation may be particularly important for patients already receiving pharmaceutical antiplatelet therapies who may become infected.
 As a result, the use of other therapeutics such as dietary sources of n-3 fatty acids or nutraceuticals with antiplatelet activities may be beneficial and warrant further investigation.
-A new clinical trial [@clinicaltrials:NCT04412018] is currently recruiting COVID-19 positive patients to investigate the anti-inflammatory activity of a recently developed, highly purified derivative of EPA known as icosapent ethyl (Vascepa ^TM^) [@doi:10.1056/NEJMoa1812792].
+A new clinical trial [@clinicaltrials:NCT04412018] is currently recruiting COVID-19 positive patients to investigate the anti-inflammatory activity of a recently developed, highly purified derivative of EPA known as icosapent ethyl (Vascepa ^TM^) [@doi:10.1056/NEJMoa1812792].
```
Comment (cbrueffer) [inline content/20.treatments.md:453]:
```suggestion
A new clinical trial [@clinicaltrials:NCT04412018] is currently recruiting COVID-19 positive patients to investigate the anti-inflammatory activity of a recently developed, highly purified derivative of EPA known as icosapent ethyl (Vascepa ^TM^) [@doi:10.1056/NEJMoa1812792].
```

File: content/20.treatments.md [hunk 2]
```diff
@@ -843,19 +843,26 @@ These cells are isolated from the patient, grown and transfected _ex vivo_, and
...
+mRNA-1273 was the first COVID-19 vaccine to enter a phase I clinical in the United States.
 ModernaTX, Inc. is currently spearheading an investigation on the immunogenicity and reactogenicity of mRNA-1273, a conventional lipid nanoparticle encapsulated RNA encoding a full-length prefusion stabilized spike (S) protein for SARS-CoV-2 [@clinicaltrials:NCT04283461].
-Forty-five participants will be enrolled in the study and given an intramuscular injection of mRNA-1273 in their deltoid muscle on Day 1 and Day 29, and then followed for the next twelve months.
-Healthy males and non-pregnant females aged 18-55 years are being recruited and will be divided in three dosage groups receiving either 25 micrograms, 100 mcg, or 250 mcg of the vaccine.
+In a study that is ongoing, forty-five participants were enrolled and given intramuscular injections of mRNA-1273 in their deltoid muscle on day 1 and day 29, and then followed for the next twelve months.
+Healthy males and non-pregnant females aged 18-55 years were recruited for this study and divided into three groups receiving either 25 micrograms, 100 mcg, or 250 mcg of mRNA-1273.
```
Comment (cbrueffer) [inline content/20.treatments.md:849]:
```suggestion
Healthy males and non-pregnant females aged 18-55 years were recruited for this study and divided into three groups receiving either 25 micrograms (mcg), 100 mcg, or 250 mcg of mRNA-1273.
```

**Comment by rando2** [comment]
(I am the one editing @cgreene's PR to add suggested reviewers -- sorry Casey!)

**Comment by cgreene** [comment]
Did anyone want to take another gander at this?

**Comment by cbrueffer** [comment]
Looks good to me!

**Pull request merged**